src/main/java/core/basesyntax/WorkWithFile.java

```diff
@@ -1,7 +1,48 @@
 package core.basesyntax;
 
+import java.io.BufferedReader;
+import java.io.BufferedWriter;
+import java.io.FileReader;
+import java.io.FileWriter;
+import java.io.IOException;
+
 public class WorkWithFile {
     public void getStatistic(String fromFileName, String toFileName) {
+        int []totals = {0,0};
```

Comment:
```suggestion
        int[] totals = {0,0};
```
format your files with IDEA's autoformatter (Ctrl Alt L) before committing